src/test/java/com/microsoft/sqlserver/jdbc/timeouts/TimeoutTest.java

```diff
@@ -0,0 +1,61 @@
+package com.microsoft.sqlserver.jdbc.timeouts;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.SQLTimeoutException;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+
+@RunWith(JUnitPlatform.class)
+public class TimeoutTest extends AbstractTest {
+    @Test
+    public void testBasicQueryTimeout() {
+        boolean exceptionThrown = false;
+        try {
+            // wait 1 minute and timeout after 10 seconds
+            Assert.assertTrue("Select succeeded", runQuery("WAITFOR DELAY '00:01'", 10));
+        } catch (SQLException e) {
+            exceptionThrown = true;
+            Assert.assertTrue("Timeout exception not thrown", e.getClass().equals(SQLTimeoutException.class));
+        }
+        Assert.assertTrue("No sql exception was thrown, a sql timeout exception should of been thrown",

```

Comment:
Please change to `No SQLException was thrown, a SQLTimeoutException was expected`

Reply:
done